docs/tutorials/index.rst

```diff
@@ -20,6 +20,11 @@ KSQL Basics
     - :ref:`ksql_quickstart-docker`
     - :ref:`ksql_quickstart-local`
 
+KSQL Recipes
+    Confluent's `KSQL Stream Processing Cookbook <https://www.confluent.io/product/ksql/stream-processing-cookbook>`__
+    includes a spectrum of pre-built stream processing sequences that you can
```

Comment:
this sentence is very awkward: `...spectrum of pre-built stream processing sequences...`. how about:

```
The KSQL Stream Processing Cookbook <https://www.confluent.io/product/ksql/stream-processing-cookbook>`__ provides in-depth tutorials and recommended deployment scenarios. 
.
```